--- mbswplspreprocess.c.orig	2014-04-16 12:54:45.000000000 -0700
+++ mbswplspreprocess.c	2014-04-18 09:33:01.000000000 -0700
@@ -21,8 +21,8 @@
  * Currently, the following procedures are offered (all optional):
  *
  * 1. Split each transducer channel into its own file. This is the only
- *practical
- * way to handle all of the various ping modes and transducer configurations.
+ * practical way to handle all of the various ping modes and transducer 
+ * configurations.
  *
  * 2. Strip rejected samples from pings. In some aquisition configurations,
  * the BathySwath records thousands of samples per ping, rejects them all,
@@ -173,10 +173,10 @@
 	"-H        print this help text\n"
 	"-Iinfile  SXP file to process\n"
 	"-Jproj4   Proj4 projection command\n"
-	"-N        do not write output to file, mostly usefull with -G\n"
+	"-N        do not write output to file, mostly useful with -G\n"
 	"-Ooutfile basename for output files [default: same as input]\n"
 	"-R        remove rejected samples from pings.\n"
-	"-S        split each transducer channel into a seperate file\n"
+	"-S        split each transducer channel into a separate file\n"
 	"-V        verbosity\n"	
 	"\n"
 	"Report bugs to the MB System development team\n";
